### ISSUE TITLE
Examples in help

### ISSUE THREAD

**Issue by alegione** [gamcil/clinker#41]
Finally got a chance to try this out, really nice tool! Thanks for all the work on it.
This isn't so much an issue as a suggestion: 
The help could use some information about default settings used or examples (e.g. 0.3 for identity).

I could just be an idiot, but when I read identity I typed a whole number (i.e. 70) just to test out the tool, and it ended with no errors (just gave an image with no alignments found...because it was looking for 700% identity!!). As I say, perhaps I should have defaulted my brain to a decimal, but wasn't sure what to input after looking at the help/usage output and I imagine others will do the same thing.

I put a few suggested additions below, happy to submit a pull request for these if it helps, but thought I'd mention it here first.

```
Alignment options:
  -na, --no_align       Do not align clusters
  -i IDENTITY, --identity IDENTITY
                        Minimum alignment sequence identity [default: 0.3]
  -j JOBS, --jobs JOBS  Number of alignments to run in parallel (0 to use the
                        number of CPUs [default])

Output options:
  -s SESSION, --session SESSION
                        Path to clinker session
  -ji JSON_INDENT, --json_indent JSON_INDENT
                        Number of spaces to indent JSON [default: none]
  -f, --force           Overwrite previous output file
  -o OUTPUT, --output OUTPUT
                        Save alignments to file [example: alignments.txt]
  -p [PLOT], --plot [PLOT]
                        Plot cluster alignments using clustermap.js. If a path
                        is given, clinker will generate a portable HTML file
                        at that path. Otherwise, the plot will be served
                        dynamically using Python's HTTP server. [example: plot.html]
  -dl DELIMITER, --delimiter DELIMITER
                        Character to delimit output by [default: tab?]
  -dc DECIMALS, --decimals DECIMALS
                        Number of decimal places in output [default: 2]
  -hl, --hide_link_headers
                        Hide alignment column headers
  -ha, --hide_aln_headers
                        Hide alignment cluster name headers

Visualisation options:
  -ufo, --use_file_order
                        Display clusters in order of input files
```

**Comment by gamcil** [comment]
Thanks for mentioning this, I've had a few people trip up on the identity being decimal vs percentage so you aren't alone haha. Just merged https://github.com/gamcil/clinker/pull/43 into master which adds some default values and more usage examples to the help menu so you can try pulling the newest version from git, otherwise it'll be in a new release sometime soon.

**Comment by alegione** [comment]
Looks great, thanks for updating, and for the great tool!

**Comment by gamcil** [comment]
Closing this since it's included in version 0.0.14, which is now released :)
